s3/compare-credentials.sh

```diff
@@ -0,0 +1,16 @@
+# Pass the credential file as an arguments
+main() {
```

Comment:
We should delete this file. As I mentioned, we don't need to include `aws-vault` commands in our shell scripts. Instead, we will use `aws-vault` to run the script and it will inject credentials into the environment variables when the shell script runs:

```
aws-vault exec <profile> -- bash s3/compare-credentials.sh
```

Reply:
Agreed, updated it to use no more vault as the user can just go into a vault environment at want